api/testutils/filtertest/runfilter.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package filtertest_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/kio"
+)
+
+func RunFilter(t *testing.T, input string, f kio.Filter) string {
```

Comment:
consider making f var args so multiple filters can be passed in (e.g. for formatting afterward)